config/nrfconnect/chip-gn/args.gni

```diff
@@ -20,8 +20,8 @@ chip_device_platform = "nrfconnect"
 
 chip_build_tests = false
 
-chip_project_config_include = ""
-chip_system_project_config_include = ""
+chip_project_config_include = "<CHIPProjectConfig.h>"
```

Comment:
Why is this necessary? These arguments are optional and we only set them if the corresponding Kconfig option is specified. What if a project does not provide the project config?